packages/office-ui-fabric-react/src/components/Nav/Nav.scss

```diff
@@ -5,196 +5,192 @@ $hasExpandButtonLinkLeftPadding: 28px;
+    display: block;
+    width: $hasExpandButtonLinkLeftPadding;
+    height: $navnode-height;
+    position: absolute;
+    top: 0;
+    @include left(0);
+    z-index: 1;
+    padding: 0;
+    margin: 0;
 
-    &.is-expanded {
-      .ms-Nav-chevron {
-        transform: rotate(-180deg);
-      }
+  .chevron {
+      position: relative;
     }
+  }
 
-    &.is-selected {
-      & a,
-      & .ms-Nav-chevronButton {
-
-        color: $ms-color-themePrimary;
-        background: $selectedBackgroundColor;
+  & a {
+  color: $unselectedForegroundColor;
+  }
+  color: $unselectedForegroundColor;
+  background: $unselectedBackgroundColor;
 
-        &:after {
-          @include border-left(2px, solid, $ms-color-themePrimary);
+  &:hover {
 
-          content: '';
-          position: absolute;
-          top: 0;
-          @include right(0);
-          bottom: 0;
-          @include left(0);
-        }
-      }
+    & a,
+    & .chevronButton {
+      background: $unselectedHoverBackgroundColor;
+      color: $unselectedHoverForegroundColor;
     }
   }
 
-  .ms-Nav-link {
-    display: block;
-    position: relative;
-    height: $navnode-height;
-    line-height: $navnode-height;
-    text-decoration: none;
-    padding: 0 20px;
-    cursor: pointer;
-    text-overflow: ellipsis;
-    text-decoration: none;
-    white-space: nowrap;
-    overflow: hidden;
+  &.compositeLinkIsExpanded {
+    .chevron {
+      transform: rotate(-180deg);
+    }
   }
 
-  .ms-Nav-groupHeaderFontSize {
-    @include ms-font-l;
-  }
+  &.compositeLinkIsSelected {
+    & a,
+    & .chevronButton {
 
-  .ms-Nav-chevronButton,
-  .ms-Nav-chevronButton--group,
-  .ms-Nav-chevronButton--link,
-  .ms-Nav-link {
-    @include focus-border();
-  }
-
-  .ms-Nav-linkButton.ms-Button--command {
-    @include padding-left($noExpandButtonLinkLeftPadding);
-    .ms-Button-label {
-      @include ms-font-m;
       color: $ms-color-themePrimary;
-    }
+      background: $selectedBackgroundColor;
+
+      &:after {
+        @include border-left(2px, solid, $ms-color-themePrimary);
 
-    &.isOnExpanded {
-    @include padding-left($hasExpandButtonLinkLeftPadding);
+        content: '';
+        position: absolute;
+        top: 0;
+        @include right(0);
+        bottom: 0;
+        @include left(0);
+      }
     }
   }
-
+}
+
+.link {
+  display: block;
+  position: relative;
+  height: $navnode-height;
+  line-height: $navnode-height;
+  text-decoration: none;
+  padding: 0 20px;
+  cursor: pointer;
+  text-overflow: ellipsis;
+  text-decoration: none;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+.groupHeaderFontSize {
+  @include ms-font-l;
+}
+
+.chevronButton,
+.chevronButtonGroup,
+.chevronButtonLink,
+.link {
+  @include focus-border();
+}
+
+:global(.ms-Nav-LinkButton.ms-Button--command) {
+  @include padding-left($noExpandButtonLinkLeftPadding);
+  :global(.ms-Button-label) {
+    @include ms-font-m;
+    color: $ms-color-themePrimary;
+  }
+
+  &.isOnExpanded {
```

Comment:
No worries, this class is in the `:global()` scope. GitHub just isn't showing it clearly in the snippet above. 😄